packages/orchestration/src/exos/chain-hub.js

```diff
@@ -432,6 +444,17 @@ export const makeChainHub = (zone, agoricNames, vowTools) => {
 
       return lookupChainInfo(chainName);
     },
+    /** @param {string} chainId */
+    getChainInfoByChainId(chainId) {
+      // Either from registerChain or memoized remote lookup()
+      chainIdToChainName.has(chainId) ||
+        Fail`Chain Info not found for ${q(chainId)}`;
+      const chainName = chainIdToChainName.get(chainId);
+      chainInfos.has(chainName) || Fail`Chain Info not found for ${q(chainId)}`;
+      return /** @type {ActualChainInfo<chainName>} */ (
```

Comment:
does it make sense that `getChainInfo` returns `ActualChainInfo<K>`, while this returns `Vow<ActualChainInfo<K>>`?